explainer-server/app/controllers/ExplainerApiImpl.scala

```diff
@@ -102,6 +102,18 @@ class ExplainerApiImpl(
     })
   }
 
+  override def delete(id: String): Future[Unit] = {
+    getStatus(id).map( publicationStatus => {
+      if (publicationStatus == Draft || publicationStatus == TakenDown) {
+        explainerDB.load(id).map( explainer => {
+          explainerDB.delete(id)
+          sendKinesisEvent(explainer, s"Sending delete event for explainer $id to PREVIEW kinesis.", previewAtomPublisher, EventType.Takedown)
```

Comment:
@philmcmahon There is no EventType delete in the thrift model (https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/contentatom.thrift#L69). Do we need to add one?